docs/releases.rst

```diff
@@ -5,6 +5,15 @@
 ======================
 
 
+tmt-1.44.0
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Execute tmt option ``--ignore-duration`` makes tmt to execute
+the test as long as it needs. Execute plugin doesn't need to be
+specified on the commandline for :ref:`plugin-variables` to work
+for this option.
```

Comment:
Shouldn't we explicitly mention the name of the environment variable which can be used to tweak this? I think those long names are not quite obvious to everybody.

Reply:
It was there, I opposed: https://github.com/teemtee/tmt/pull/3569#discussion_r1980884950

Reply:
I'll keep it as is.

Reply:
Ok.